packages/size-limit/create-reporter.js

```diff
@@ -167,7 +168,8 @@ function createHumanReporter (process, isSilentMode = false) {
       }
 
       if (config.failed) {
-        let fix = 'Try to reduce size or increase limit'
+        let fix =
+          'Try to reduce size, increase limit or specify correct file path'
```

Comment:
The missed path is a rare case. We should show `specify corrent file path` only if we had this type of error (we can add `config.missed`)

Reply:
You're right